databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/discovery/EmoServiceDiscovery.java

```diff
@@ -0,0 +1,11 @@
+package com.bazaarvoice.emodb.databus.client2.discovery;
+
+
+import java.net.URI;
+import java.net.UnknownHostException;
+
+//TODO add documentation and tests
```

Comment:
TODO

Reply:
TODO will be tracked and resolved under [PD-174014](https://bazaarvoice.atlassian.net/browse/PD-174014)